instagram/src/components/PostContainer/PostCard.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component} from 'react'
+import CommentCard from '../CommentSection/CommentCard';
+
+class PostCard extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      likes: props.data.likes
+    };
+  }
+
+  // handleToggleLike = (text) => {
+    
+  //   // let likes = this.state.likes;
+    
+  //   this.setState(prevState => (
+  //     {
+  //       likes: prevState.likes.map(post => {
+  //         if (text === post.text) {
+  //           return {
+  //             ...post,
+  //             likes: post.likes + 1
+  //           }
+  //         }
+  //       })
+  //     }
+  //   ))
+  // }
+
+  
+  toggleLike = (e) => {
```

Comment:
Like handler is nice (I like how you can only add 1 like, like in real life). The color change is clean.